earn/src/components/portfolio/modal/SendCryptoModal.tsx

```diff
@@ -200,6 +207,9 @@ export default function SendCryptoModal(props: SendCryptoModalProps) {
     setSelectedOption(defaultOption);
   }, [defaultOption]);
 
+  const bigSendAmount = new Big(sendAmountInputValue || 0).mul(String1E(selectedOption.decimals));
+  const bigSendBalance = new Big(depositBalance?.formatted ?? 0).mul(String1E(selectedOption.decimals));
```

Comment:
Does `depositBalance` not have a `BigNumberish` property that we can convert from? I'd rather use that than the `formatted` thing, because we could avoid having to multiple by the `String1E` decimals value.

Reply:
Acked, but holding off until we do the GoodNumber update.